manifests/init.pp

```diff
@@ -122,8 +122,8 @@
   Optional[Variant[Array[String], String]] $http_raw_append  = undef,
   Enum['on', 'off'] $http_tcp_nodelay                        = 'on',
   Enum['on', 'off'] $http_tcp_nopush                         = 'off',
-  $keepalive_timeout                                         = '65s',
-  $keepalive_requests                                        = '100',
+  String $keepalive_timeout                                  = '65s',
+  String $keepalive_requests                                 = '100',
```

Comment:
Even the the default is (questionably), a string, did Integers also use to work here?

Reply:
The docs have `keepalive_requests` down as a `number`, so we should definitely be allowing `Integer` here.

Not sure if suffixes are allowed on `numbers`.  eg. `'1k'` instead of `1000`.  Can't see this in the docs, but would have to test to be 100% sure.

Reply:
@bastelfreak This one definitely needs updating for this PR.  People _will_ have used `Integers` (which is correct for something nginx describes as a `number`).

Reply:
Tried to set it to `1k`, it fails with:

```
nginx: [emerg] "keepalive_requests" directive invalid number in /etc/nginx/nginx.conf:36
```

Let's use an Integer!